exporter/opentelemetry-exporter-datadog/tests/test_datadog_exporter.py

```diff
@@ -245,8 +255,13 @@ def test_translate_to_datadog(self):
                 start=start_times[2],
                 duration=durations[2],
                 error=0,
-                service="test-service",
-                meta={"env": "test", "team": "testers", "version": "0.0.1"},
+                service="resource_service_name",
```

Comment:
Also want a test for when the `span.resources` does not include `service.name` and it's backed off to the existing configuration.

Reply:
added 👍